sphinx_scality/footer.html

```diff
@@ -5,7 +5,7 @@
     <div class="baselogo">
       {%- if has_homelink %}<a href="{{ theme_homelink }}">{% endif %}
       <img class="baselogo"
-           src="{{ pathto('_static/img/scality-documentation-white.svg', 1) }}"
+           src="{{ pathto('_static/img/scality-documentation-white.png', 1) }}"
```

Comment:
Don't we have them available in SVG?

Reply:
All the logos I was given are in .png 😢 I can ask marketing if they are available in .svg, though.